boards/arm/adafruit_feather_m0/adafruit_feather_m0_lora.yaml

```diff
@@ -0,0 +1,10 @@
+identifier: adafruit_feather_m0_lora
+name: Adafruit Feather M0 LoRa
+type: mcu
+arch: arm
+ram: 32
+flash: 256
```

Comment:
Either way I think this is no different than was already the case for the basic_proto variant (see https://github.com/zephyrproject-rtos/zephyr/blob/1d879b93c59e8f1f48d86db5794600769c86facb/boards/arm/adafruit_feather_m0_basic_proto/adafruit_feather_m0_basic_proto.yaml ) .

Should we open a separate issue to fix for both variants ?

Reply:
Since you are renaming here I think you can just create another commit after to fix it.
If you prefer, you could split into 3 commits to make clearly:
* preparation (the renaming to track history)
* your changes
* the fix